tests/server/test_env_settings.py

```diff
@@ -38,6 +38,17 @@ def check_only_contains_default_setting(settings_dict: Dict[str, object]) -> Non
         assert setting_value == get_environment_setting_default(setting_name)
 
 
+async def test_api_return_type(client, server, environment_default):
```

Comment:
Why does this test case use the `environment_default` fixture instead of the `environment` fixture we always use?

Reply:
Ah, I copy pasted the test definition from `test_environment_settings`. Fixed to use the `environment` fixture.
I wonder, what would the issue be with using `environment_default` instead ? I'm guessing it is the fact that auto_deploy is true in that one ? Could it cause flakiness/slowness ?